snapshot-labs/sx-monorepo
# refactor: modernize Select-type inputs

### Summary

This PR modernizes some of the Select-type inputs:
- Now they are typed so you pass necessary info (`options` and `enum`).
- Clarified when different selects are used (`Select` vs `SelectMulti`) - `SelectMulti` will now be used every time array input is used with enum, regardless of maxItems.
- SelectMulti now floats, same as Combobox (so it can be used in modals).
- Removed broken `InputArray`.

This PR was extracted from code to add support for validations which pulls validation from server and definitions include different selects.

### How to test

1. Go to different Select/SelectMulti/Combobox we have in the app.
2. They all work as expected.
